pyproject.toml

```diff
@@ -25,22 +25,18 @@ python = "^3.8"
 typer = {extras = ["all"], version = "^0.7.0"}
 qutechopenql = "^0.11.1"
 pydantic = "^1.10.7"
+qi-compute-api-client = {git = "https://github.com/QuTech-Delft/compute-api-client.git"}
```

Comment:
Should we not pin this? It could unexpectedly use a newer version when releasing vs when the last change was made.